projects/aca-content/src/lib/components/trashcan/trashcan.module.ts

```diff
@@ -27,30 +27,17 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { CoreModule } from '@alfresco/adf-core';
 import { TrashcanComponent } from './trashcan.component';
-import { Routes, RouterModule } from '@angular/router';
 import { ContentModule } from '@alfresco/adf-content-services';
 import { AppCommonModule } from '../common/common.module';
 import { AppToolbarModule } from '../toolbar/toolbar.module';
 import { DirectivesModule } from '../../directives/directives.module';
 import { ContextMenuModule } from '../context-menu/context-menu.module';
 import { AppLayoutModule } from '../layout/layout.module';
 
-const routes: Routes = [
-  {
-    path: '',
-    component: TrashcanComponent,
-    data: {
-      title: 'APP.BROWSE.TRASHCAN.TITLE',
-      sortingPreferenceKey: 'trashcan'
-    }
-  }
-];
-
```

Comment:
The same as [here](https://github.com/Alfresco/alfresco-content-app/pull/2859/files#r1065701766)